src/Compilers/CSharp/Test/Syntax/Parsing/LambdaAttributeParsingTests.cs

```diff
@@ -2202,58 +2202,59 @@ public void NullableType_Is_01()
             {
-                N(SyntaxKind.IdentifierName);
+                N(SyntaxKind.IsExpression);
                 {
-                    N(SyntaxKind.IdentifierToken, "x");
+                    N(SyntaxKind.IdentifierName);
+                    {
+                        N(SyntaxKind.IdentifierToken, "x");
+                    }
+                    N(SyntaxKind.IsKeyword);
+                    N(SyntaxKind.PredefinedType);
+                    {
+                        N(SyntaxKind.StringKeyword);
+                    }
                 }
-                N(SyntaxKind.IsKeyword);
-                N(SyntaxKind.DeclarationPattern);
+                N(SyntaxKind.QuestionToken);
+                N(SyntaxKind.CollectionExpression);
                 {
-                    N(SyntaxKind.ArrayType);
+                    N(SyntaxKind.OpenBracketToken);
+                    N(SyntaxKind.ExpressionElement);
                     {
-                        N(SyntaxKind.NullableType);
-                        {
-                            N(SyntaxKind.PredefinedType);
-                            {
-                                N(SyntaxKind.StringKeyword);
-                            }
-                            N(SyntaxKind.QuestionToken);
-                        }
-                        N(SyntaxKind.ArrayRankSpecifier);
+                        N(SyntaxKind.IdentifierName);
                         {
-                            N(SyntaxKind.OpenBracketToken);
-                            N(SyntaxKind.IdentifierName);
-                            {
-                                N(SyntaxKind.IdentifierToken, "A");
-                            }
-                            N(SyntaxKind.CloseBracketToken);
+                            N(SyntaxKind.IdentifierToken, "A");
                         }
                     }
-                    N(SyntaxKind.SingleVariableDesignation);
-                    {
-                        N(SyntaxKind.IdentifierToken, "y");
-                    }
+                    N(SyntaxKind.CloseBracketToken);
+                }
+                M(SyntaxKind.ColonToken);
+                N(SyntaxKind.IdentifierName);
+                {
+                    N(SyntaxKind.IdentifierToken, "y");
                 }
             }
             EOF();
         }
 
-        [Fact]
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/75318")]
         public void NullableType_Is_03()
         {
             string source = "_ = x is string ? [] y => y : z";
             UsingExpression(source,
-                // (1,1): error CS1073: Unexpected token '=>'
+                // (1,20): error CS1001: Identifier expected
                 // _ = x is string ? [] y => y : z
-                Diagnostic(ErrorCode.ERR_UnexpectedToken, "_ = x is string ? [] y").WithArguments("=>").WithLocation(1, 1));
+                Diagnostic(ErrorCode.ERR_IdentifierExpected, "]").WithLocation(1, 20));
```

Comment:
this parsing matches teh parsing of hte test below this one (`"_ = x is string ? [A] y => y : z"`).  Specifically, we now see this *appropriately* as a conditional that returns a lambda.  But we report an issue on the lambda that it's attribute-list is missing attributes.  This is  better parse than before, more in line with the parse once you add the attribute.